src/pages/add-product/components/image-list/styles.ts

```diff
@@ -0,0 +1,33 @@
+import styled from '@emotion/styled';
```

Comment:
It is preferable to use 
```suggestion
import { styled } from "@mui/material";
```
for style